scalableminds/webknossos
# Allow selective segment visibility regardless of proofreading tool

### URL of deployed dev instance (used for testing):
- https://selectivesegmentvisibility.webknossos.xyz

### Steps to test:
- view annotations with a segmentation layer
- ensure that the hovering of segments works as usual
- enable the new "Selective Visibility" toggle in the sidebar -> only the active segment and hovered segments should be visible
- switch to the proofreading mode -> the selective visibility toggle should be disabled now. the already existing selective visibility toggle in the toolbar should now have precedencw

### Issues:
- fixes #8260 

------
(Please delete unneeded items, merge only when none are left open)
